test/stdcpp/src/allocator.cpp

```diff
@@ -0,0 +1,12 @@
+#include <memory>
+
+//Anything that will instantiate an allocator
+#include <vector>
+
+struct Mystruct { int a; };
+void foo()
+{
+    std::vector<int> a = { 10 };
```

Comment:
this doesn't seem to really instantiate the full allocator, `template std::allocator<int>` does.

Reply:
It worked for me ;) ... i'll change it.

Reply:
Yeah we had [this problem before](https://github.com/dlang/dmd/pull/9136)